lib/berkeley_library/av/core/module_info.rb

```diff
@@ -3,12 +3,12 @@ module AV
     module Core
       class ModuleInfo
         NAME = 'berkeley_library-av-core'.freeze
-        AUTHOR = 'David Moles'.freeze
-        AUTHOR_EMAIL = 'dmoles@berkeley.edu'.freeze
+        AUTHORS = ['David Moles', 'maría a. matienzo'].freeze
+        AUTHOR_EMAILS = ['dmoles@berkeley.edu', 'matienzo@berkeley.edu'].freeze
```

Comment:
Normally you want to freeze strings, not arrays.  The array is now immutable, but the string isn't.

Might be simpler to add `# frozen_string_literal: true` to the top of the file and remove these?